crates/sui-framework/sources/governance/delegation.move

```diff
@@ -107,6 +107,40 @@ module sui::delegation {
         self.ending_epoch = option::some(ending_epoch);
     }
 
+    /// Switch the delegation from the current validator to `new_validator_address`.
+    /// The current `Delegation` object `self` becomes inactive and the balance inside is
+    /// extracted to the new `Delegation` object. 
+    public(friend) fun switch_delegation(
```

Comment:
IIRC we agreed that switching shouldn't have a delay so that delegators can switch away from a malicious validator as soon as possible. cc @econmysten for confirmation

Reply:
yes confirmed